Formula/gitversion.rb

```diff
@@ -1,27 +1,31 @@
 class Gitversion < Formula
   desc "Easy semantic versioning for projects using Git"
-  homepage "https://github.com/GitTools/GitVersion"
-  url "https://github.com/GitTools/GitVersion/releases/download/5.0.1/GitVersion-bin-fullfx-v5.0.1.zip"
-  sha256 "9b543d3e42e0d5e6fab0b44553cb6bbbb0e31431030ef761fc1a50c845fd166a"
+  homepage "https://gitversion.net"
+  url "https://github.com/GitTools/GitVersion/archive/5.3.7.tar.gz"
+  sha256 "70c6f867e4a85325ba1e54211fd014d9b3ec9be43bb828393e1f0d6a5e33cefb"
   license "MIT"
 
-  bottle :unneeded
+  depends_on "dotnet"
 
-  depends_on "mono"
+  def install
+    system "dotnet", "build",
+           "--configuration", "Release",
+           "--framework", "netcoreapp3.1",
+           "--output", "out",
+           "src/GitVersionExe/GitVersionExe.csproj"
 
-  uses_from_macos "icu4c"
```

Comment:
Does it not need this anymore?

Reply:
Seems not.

Reply:
I removed it since `dotnet` now provides this dependency.

https://github.com/Homebrew/homebrew-core/blob/56199837ec6d880e038a38983afd8f2369b0882b/Formula/dotnet.rb#L20

Reply:
If its needed by both thr formula and its dependencies i believe we should still depend on it explicitly.

Reply:
I don't know why GitVersion would need `icu4c` itself as it only relies on .NET to handle i10n.